uint8 PLANE = 0

Header header
float64[4] coef
float64[3] normal
geometry_msgs/Point center
geometry_msgs/Vector3 min
geometry_msgs/Vector3 max
geometry_msgs/Point32[] polygon
std_msgs/Int32 size
bool is_vertical
